redhat-cop/containers-quickstarts
# ubi8-asciidoctor: Update package versions & increment major version

#### What is this PR About?
The biggest update is moving to the 2.x version of ascidoctor-pdf. This
may cause breakages as defining the theme has many updates.

A few minor typo's are also fixed in the README.

I incremented the major version due to the change to Ruby 3.0 and Asciidoctor-PDF 2.2.0

#### How do we test this?

1. Build a test container using the Dockerfile
2. Run the container and check that the commands such as asciidoctor-pdf work.

cc: @redhat-cop/day-in-the-life
